flake.nix

```diff
@@ -41,7 +41,7 @@
     };
 
     blockfrost.url = "github:mlabs-haskell/blockfrost-backend-ryo/aciceri/nix-flake";
-    db-sync.url = "github:mlabs-haskell/cardano-db-sync/aciceri/customConfig-input-fix";
+    db-sync.url = "github:input-output-hk/cardano-db-sync/13.1.0.0";
```

Comment:
Why not update our fork? It's not a change request, but a question

Reply:
I tried to do it with the original flake, didn't quite work.
Do we have any other change in our fork except customConfig?